lib/database_consistency/helper.rb

```diff
@@ -34,5 +34,39 @@ def first_level_associations(model)
 
       associations
     end
+
+    # @return [Array<String>]
+    def extract_index_columns(index_columns)
+      return index_columns unless index_columns.is_a?(String)
+
+      index_columns.split(',')
+                   .map(&:strip)
+                   .map { |str| str.gsub(/lower\(/i, 'lower(') }
+                   .map { |str| str.gsub(/\(([^)]+)\)::\w+/, '\1') }
+                   .map { |str| str.gsub(/'([^)]+)'::\w+/, '\1') }
+    end
+
+    def sorted_uniqueness_validator_columns(attribute, validator, model)
+      uniqueness_validator_columns(attribute, validator, model).sort
+    end
+
+    def uniqueness_validator_columns(attribute, validator, model)
```

Comment:
The three methods below could be made private, but I didn't want to make too many changes to this module